esmvaltool/diag_scripts/droughts/spei.R

```diff
@@ -0,0 +1,190 @@
+# DESCRIPTION:
+#
+# This file is based on code from diag_save_spei_all.R and should replace
+# diag_spei.R at some point. It is more flexible and provides the same
+# functionality as the previous diag_spei.R as a special case.
+# Calculation of PET and plotting of results is covered in seperate diagnostics.
+#
```

Comment:
```suggestion
```
Again I think better in changelog or documentation than in the file where nobody will look for it =D